docs/book/src/cronjob-tutorial/testdata/finalizer_example.go

```diff
@@ -68,23 +68,23 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		// The object is being deleted
```

Comment:
Also, the example is missing the import. 

```
	batch "k8s.io/api/batch/v1beta1"
```

Reply:
I think the issue here is not that it is missing that, it is that the `batchv1` alias is wrong, it should be `batch`.

Reply:
See [here](https://github.com/kubernetes-sigs/kubebuilder/blob/cb877e9f82a168c7ac80cee1ee5e194e63d8ed18/docs/book/src/cronjob-tutorial/testdata/finalizer_example.go#L24) that the import is not in the k8s API for CronJobs is not in the example @Adirio

Reply:
But this is the reconciler for `tutorial.kubebuilder.io/project/api/v1.CronJob` not for `k8s.io/api/batch/v1beta1.CronJob`. Remember this tutorial is creating a clone of the builtin CronJob to provide an example of how a controller can be developed.

Reply:
Make sense. So we need to fix the alias and replace the `batch` for `batchv1`.

Reply:
Exactly